### ISSUE TITLE
Training notebook `tpv13` does not run inside docker container

### ISSUE THREAD

**Issue by SamNewcome** [SeisSol/Training#41]
Training notebook `tpv13` does not run inside docker container. Specifically this is the visualisation with Paraview part.

Here the command is
~~~
docker cp my_docker:/home/training/tpv13/my_output .
~~~

Adding the `!` needed for calling the bash script:
~~~
docker cp my_docker:/home/training/tpv13/my_output .
~~~
gives output:
~~~
/bin/bash: line 1: docker: command not found
~~~

Maybe the docker container does not include docker?

**Comment by sebwolf-de** [comment]
Thanks for the report.
I guess this needs some clarification in the notebook. You are meant to run this on your host computer.

**Comment by sebwolf-de** [comment]
@SamNewcome Please check if the image `seissol/training:pr-42` fixes your issue? The image will be available, once the CI of PR#42 has finished.
